students/nam_vo/lesson02/series.py

```diff
@@ -0,0 +1,51 @@
+def fibonacci(n):
```

Comment:
Innovative solution to base both fibonacci and lucas functions on sum_series - first time I've seen it!